### ISSUE TITLE
Unstandardized Aesthetic's method names

### ISSUE THREAD

**Issue by HugoCastelani** [afollestad/aesthetic#10]
Just a little detail, method `primaryTextColorInverseRes` invert "Inverse" and "Color" order.

**Comment by afollestad** [comment]
I don't think it's unstandardized. The equivalent Android attribute is `android.R.attr.textColorPrimaryInverse`

**Comment by afollestad** [comment]
I might actually rename the methods to be like `textColorPrimary`, etc.

**Comment by HugoCastelani** [comment]
Hm... Seems a bit weird to me:

```
.primaryTextInverseColor()
.primaryTextColorInverseRes()
.secondaryTextInverseColor()
.secondaryTextInverseColorRes()
```

**Comment by afollestad** [comment]
Ah, I see what you mean.

**Comment by afollestad** [comment]
I've changed the naming conventions of all the methods, now available in 0.1.0.

For an example, text color properties start with `textColor`, color properties start with `color` such as `colorPrimary`, etc.

**Comment by HugoCastelani** [comment]
Excellent.
